components/CoverletterInformation.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <address>
+    <div 
+      class="company">
+      <span 
+        v-if="title"
+        v-text="title"/>
+    </div>
+    <div 
+      class="department">
+      <span 
+        v-if="department"
+        v-text="department"/>
+    </div>
+    <div 
+      class="name">
+      <span 
```

Comment:
Instead of outputting everything explicitly you could also just loop through your name object and output every property since for loops in vue also work with objects. this will also enable you to savely handle "non existence" of values without having 100s of v-if